src/jit/unwindamd64.cpp

```diff
@@ -142,6 +233,32 @@ void Compiler::unwindPush(regNumber reg)
     }
 }
 
+#ifdef UNIX_AMD64_ABI
+void Compiler::unwindPushCFI(regNumber reg)
+{
+    assert(compGeneratingProlog);
+
+    FuncInfoDsc* func = funCurrentFunc();
+
+    unsigned int cbProlog = unwindGetCurrentOffset(func);
+    noway_assert((BYTE)cbProlog == cbProlog);
```

Comment:
Code duplication is intentional to cleanly separate from Windows code per Bruce's feedback. As commented, once we introduce a unwind blob that we can virtually dispatch these things per platform/target/flavor.